ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -856,6 +861,11 @@ router bgp {{ router_bgp.as }}
       network {{ network.prefix }}
 {%             endif %}
 {%         endfor %}
+{%         if router_bgp.address_family_ipv4.bgp.redistribute_internal is arista.avd.defined(true) %}
```

Comment:
Are you sure about the command ordering, that it comes after `network` but above `redistribute` ?

Reply:
Yes. This config is from eos. bgp redistribute is after network and before aggreagate-address.

   neighbor 192.0.255.1 description s1-spine1
   neighbor 192.0.255.2 peer group EVPN-OVERLAY-PEERS
   neighbor 192.0.255.2 remote-as 65001
   neighbor 192.0.255.2 description s1-spine2
   no bgp redistribute-internal
   aggregate-address 1.1.1.0/24 advertise-only
   aggregate-address 2.2.1.0/24
   redistribute connected rcf Router_BGP_Connected()
   redistribute ospf include leaked

Reply:
Thank you for confirming!